javascript-8/javascript-8.md

```diff
@@ -77,10 +81,12 @@ When you *instantiate* a class, that means you are creating an object from a cla
 #### Encapsulation
 *Encapsulation* means that properties that make sense together are all together in a single object.  In the example above, all the information about a book are encapsulated in the `book` object and all the information about a borrower are encapsulated in the `borrower` object.  Properties about a book are not stored inside a `borrower` object, and vice versa.
 
+Methods can be part of objects too, and the principle of encapsulation also applies to methods.  For example, a method called `checkOut()` would make sense inside a `borrower` object.  It would not be relevant for a `book` object to have that method, because the `book` object would never check anything out of the library.
+
 #### Inheritance
-*Inheritance* is a way for objects to inherit properties from other objects, using a class hierarchy.
+*Inheritance* is a way for objects to inherit properties and methods from other objects, using a class hierarchy.
```

Comment:
Class hierarchy will probably confusing here without a definition

Reply:
Good call!  Okay, editing this.